### ISSUE TITLE
Apps support

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#52]
```
Request from the market: Needs apps
```

Original issue reported on code.google.com by `kaaholst` on 2012-02-14 22:49:57

**Comment by kaaholst** [comment]
```
What would this actually do?

When I use the web UI to go to apps it takes me to http://www.mysqueezebox.com/appgallery?player=[playerid]
and shows me a list of apps I can install, but the apps I install don't then seem to
show up in the web UI.

I may well be doing something wrong...
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2012-02-15 21:14:13

**Comment by kaaholst** [comment]
```
Apps support would be really handy - e.g. I use the Google Music plugin by Hechtus:
https://github.com/hechtus/squeezebox-googlemusic which appears as an App in both the
Logitech web UI and in Logitech's own Squeezebox app.

e.g. in the Logitech Squeezebox app I go Home -> My Apps -> Google Music -> All Access
-> Search - then I can enter an artist/album/track name to search Google Music and
add a track to the current playlist.

the 'My Apps' section is useful since it also gives quick access to Internet radio
networks like SomaFM and SKY.fm, plus any Podcasts I've configured.
```

Original issue reported on code.google.com by `gdh@acentral.co.uk` on 2013-12-19 22:29:06

**Comment by kaaholst** [comment]
```
I'd really like apps support too - Squeeze Commander can access Spotify from its main
menu (as I also have it set up on my Boom) and this is what I'd like in Squeezer although
I prefer Squeezer's UI in general.
```

Original issue reported on code.google.com by `lefthandandy` on 2014-02-20 17:14:58

**Comment by kaaholst** [comment]
```
Are Apps the same as LMS plugins? I'd like to control the "Smart Mix" plugin, which
works fine in Orange Squeeze. It appears in the main menu there, exactly like in the
server web interface.
I do not use anything mysqueezebox.com related though.
```

Original issue reported on code.google.com by `thenktor` on 2014-09-03 07:52:00

**Comment by kaaholst** [comment]
```
I would be interested to see apps support as well.  I too use the google music plugin
by Hechtus, as well as Pandora and Sirius.  It would be great to be able to use squeezer
to access and play that content.
```

Original issue reported on code.google.com by `mbelleville` on 2014-10-07 20:16:47

**Comment by kaaholst** [comment]
```
The same, LMS (local server) APPS is verry missing.
It's the only reason why I use the official Logitech SqueezeBox APK instead of Squeezer...

I would prefer to use Squeezer and I do as soon as Apps will be supported !

```

Original issue reported on code.google.com by `zeGeaX` on 2014-11-14 12:47:22

**Comment by kaaholst** [comment]
```
I would also greatly appreciate this feature, as I am using the third party Spotify
plugin. Searching music on spotify cannot be done right now with Squeezer. Now have
to use Squeeze commander for that, but would like to use Squeezer for everything!
```

Original issue reported on code.google.com by `gijsmob` on 2014-11-26 23:16:02

**Comment by kaaholst** [comment]
https://github.com/nikclayton/android-squeezer/wiki/Apps-support-in-Squeezer


Original issue reported on github.com/nikclayton/android-squeezer by nikclayton on 2015-06-01T11:36:51Z

**Comment by kaaholst** [comment]
Fixed in 2.0.0

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-03-07T19:30:16Z
